src/lib/rest/rest.cpp

```diff
@@ -1239,6 +1245,25 @@ static int connectionTreat
       return MHD_NO;
     }
 
+
+    // Get API version
+    // FIXME #3109-PR: this assignment will be removed in a subsequent PR, where the function apiVersionGet() is used instead
```

Comment:
What I mean is:

`FIXME PR`: something to be fixed in this same PR (typically a mark the developer uses as he implements during the PR, i.e. no FIXME PR should "survive" after review step previous to merge)
`FIXME #3109-PR`: something to be fixed before the last PR which is part of the multi-PR issue #3109

NTC (just trying to clarify)

Reply:
Fixed in 8413645